### PR TITLE
added a new category drink

### PR DESCRIPTION
**Pull request by aldinaufal21**

### CODE REVIEW

**Comment by gabrielbarker** [comment]
Hey @aldinaufal21 
This class looks good. Please could you take a look at the CONTRIBUTING.md file. It details the other steps needed to contribute a category. Namely, adding a test file that matches the other tests for categories, and adding "drink" to the __caregory_map in the pattern class. Also it asks that you include at least 10 values in a category.
Let me know if you need any help with this. Good luck!

**Comment by aldinaufal21** [comment]
okay @gabrielbarker i will add it

**Comment by gabrielbarker** [comment]
The [build](https://travis-ci.com/github/gabrielbarker/qual-id/jobs/392560038) has failed. It seems to be because you haven't imported 'Drink' at the top of the pattern class. Also you still need to add a test class.

**Comment by gabrielbarker** [comment]
[This PR](https://github.com/gabrielbarker/qual-id/pull/4) might help as a reference. Check out the Files Changed tab

**Comment by aldinaufal21** [comment]
in the pattern class there is a conflict. to finish, just delete part of the instrument code?

**Comment by gabrielbarker** [comment]
I've resolved the conflicts, we need the Instrument stuff _and_ the Drink stuff
There are now 2 test failures. They are both due to the fact that your `get_values` method has been written incorrectly. I will add a comment to show you what I mean
🙂

File: qual_id/categories/drink.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from ..category import Category
+
+class Drink(Category):
+    def get_values:
```
Comment (gabrielbarker) [inline qual_id/categories/drink.py:4]:
```suggestion
    def get_values(self):
```
Methods need brackets `()`.
Also, this is a class method, and so it needs to take the parameter `self`.

Comment (aldinaufal21) [reply qual_id/categories/drink.py:4]: Im sorry, i forgot to add it

Comment (gabrielbarker) [reply qual_id/categories/drink.py:4]: Don't worry about it! If you click 'commit suggestion' it should fix the tests

**Review by gabrielbarker — APPROVED** [review]
Nice work @aldinaufal21 !

**Comment by gabrielbarker** [comment]
I've merged the PR, and the code live [here](https://qual-id.herokuapp.com/get/?pattern=drink-animal). Thanks a lot for your contribution and congratulations on your first PR!
🎉

**Pull request merged**